server/src/test/java/com/exasol/testsetupabstraction/server/TestSetupServerTest.java

```diff
@@ -62,4 +67,38 @@ void testGET_connectionInfo() {
         get("/connectionInfo").then()
                 .body(equalTo("{\"host\":\"localhost\",\"password\":\"myPass\",\"port\":123,\"user\":\"myUser\"}"));
     }
+
+    @Test
+    void testGET_listFiles() throws BucketAccessException {
+        final Bucket bucket = mock(Bucket.class);
+        doReturn(List.of("myFile")).when(bucket).listContents("/my-path");
+        doReturn(bucket).when(testSetup).getDefaultBucket();
+        given().queryParam("path", "/my-path").get("/bfs/listFiles").then().body(equalTo("{\"files\":[\"myFile\"]}"));
+    }
+
+    @Test
+    void testPOST_uploadStringContent() throws BucketAccessException, InterruptedException, TimeoutException {
+        final Bucket bucket = mock(Bucket.class);
+        doReturn(bucket).when(testSetup).getDefaultBucket();
+        given().formParam("stringContent", "myString").formParam("remoteName", "myFile.txt")
+                .post("/bfs/uploadStringContent").then().body(equalTo("{\"ok\":true}"));
+        verify(bucket).uploadStringContent("myString", "myFile.txt");
+    }
+
+    @Test
+    void testPOST_uploadFile() throws BucketAccessException, TimeoutException, FileNotFoundException {
+        final Bucket bucket = mock(Bucket.class);
+        doReturn(bucket).when(testSetup).getDefaultBucket();
+        given().formParam("localPath", "/tmp/myFile.txt").formParam("remoteName", "myFile.txt").post("/bfs/uploadFile")
+                .then().body(equalTo("{\"ok\":true}"));
```

Comment:
doesn't java has something like `raw strings`? Reading escaped strings in test would be unnecessary hard if java has such a mechanism.